pkg/remote/remote.go

```diff
@@ -198,18 +208,20 @@ func (r *Remote) Fetch(goGetterSrc string, cacheDirOpt ...string) (string, error
 			getterSrc = u.Getter + "::" + getterSrc
 		}
 
-		r.Logger.Debugf("remote> downloading %s to %s", getterSrc, getterDst)
+		r.Logger.Debugf("remote> downloading %s to %s", getterSrc, originDirOrFilePath)
 
-		if err := r.Getter.Get(r.Home, getterSrc, cacheDirPath); err != nil {
-			rmerr := os.RemoveAll(cacheDirPath)
+		if err := r.Getter.Get(r.Home, getterSrc, originDirOrFilePath); err != nil {
+			rmerr := os.RemoveAll(originDirOrFilePath)
 			if rmerr != nil {
 				return "", multierr.Append(err, rmerr)
 			}
 			return "", err
 		}
 	}
-
-	return filepath.Join(cacheDirPath, file), nil
+	if file == "" {
+		return originDirOrFilePath, nil
+	}
```

Comment:
This judgment may not be required, 
when `file==""`, 
`filepath.Join(originDirOrFilePath, file) == originDirOrFilePath`

Reply:
got.